src/libraries/System.Text.Json/tests/Serialization/CustomConverterTests.BadConverters.cs

```diff
@@ -332,5 +332,38 @@ public static void TypeHasMoreThanOneConverter()
             Assert.Contains("'System.Text.Json.Serialization.JsonConverterAttribute'", ex.Message);
             Assert.Contains("'System.Text.Json.Serialization.Tests.CustomConverterTests+PocoWithTwoConverters'", ex.Message);
         }
+
+        [Fact]
+        public static void ConverterWithoutDefaultCtor()
+        {
+            string json = @"{""MyType"":""ABC""}";
+
+            InvalidOperationException ex = Assert.Throws<InvalidOperationException>(() => JsonSerializer.Deserialize<ClassWithConverterWithoutPublicEmptyCtor>(json));
+            Assert.Equal("The converter specified on 'System.Text.Json.Serialization.Tests.CustomConverterTests+ClassWithConverterWithoutPublicEmptyCtor' " +
+                "does not derive from JsonConverter or have a public parameterless constructor.", ex.Message);
```

Comment:
We generally don't test for the exact match on exception messages (since those can change and such tests would fail if the message was localized).

Feel free to use the existing pattern (Assert.Contains), or just assert that the exception got thrown.

```suggestion
            Assert.Contains("'System.Text.Json.Serialization.Tests.CustomConverterTests+ClassWithConverterWithoutPublicEmptyCtor'", ex.Message);
```